test/fixtures/permission/fs-write.js

```diff
@@ -130,6 +130,23 @@ const absoluteProtectedFolder = path.resolve(relativeProtectedFolder);
   }));
 }
 
+{
+  assert.throws(() => {
+    fs.mkdtempSync(path.join(blockedFolder, 'any-folder'));
+  }, common.expectsError({
+    code: 'ERR_ACCESS_DENIED',
+    permission: 'FileSystemWrite',
+  }));
+  assert.throws(() => {
+    fs.mkdtemp(path.join(relativeProtectedFolder, 'any-folder'), (err) => {
+      assert.ifError(err);
```

Comment:
should this function be also checked with common.mustNotCall?

Reply:
well... might be, but I prefer doing it in a separated PR considering all the above assertions aren't using it.